.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,36 @@
+name: Feature Request
+description: Suggest a feature for this project
+title: "[Feature]: "
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the feature you want to add
+      placeholder: >
+        I'd like to be able to [...]
+    validations:
+      required: true
+  - type: textarea
+    id: possible-solution
+    attributes:
+      label: Describe your proposed solution
+      placeholder: >
+        I think this could be done by [...]
+    validations:
+      required: false
+  - type: textarea
+    id: alternatives
+    attributes:
+      label: Describe alternatives you've considered, if relevant
+      placeholder: >
+        Another way to do this would be [...]
+    validations:
+      required: false
+  - type: textarea
+    id: additional-context
+    attributes:
+      label: Additional context
+      placeholder: >
+        Add any other context or screenshots about the feature request here.
+    validations:
+      required: false
```

Comment:
@sadrasabouri 
Please address this minor end-of-file bug.

Reply:
fixed in [5a9076a](https://github.com/sepandhaghighi/nafas/pull/89/commits/5a9076ae62e0b1eb0dd3fef15ec8c60ba6ccacd4)